server/src/main/java/org/opensearch/search/aggregations/metrics/SumNullAggregator.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
And here please

Reply:
removed in https://github.com/opensearch-project/OpenSearch/pull/6984/commits/7d50f500d045f1c8acb3d71318ef777cef6e6143